examples/docker/00-hello-world/worker.js

```diff
@@ -0,0 +1,15 @@
+const Queue = require('bee-queue');
+
+// Core worker logic
+const workerProcess = (job, done) => {
+  done(null, {greeting: 'Hello ' + job.data.name + '!'});
+};
```

Comment:
It might be preferable to make this an `async` function, as this is an example
```suggestion
async function workerProcess(job) {
  return {greeting: 'Hello ' + job.data.name + '!'};
}
```

Reply:
Yes, much preferred.  I originally had an async using fat-arrow syntax, and the linter choked on it.  These days I refuse to use the `function` keyword ;-) / 2.  Perhaps we could have more relaxed (modern) linter configuration for the examples directories?

Reply:
Heh, the linter doesn't like this syntax either!

Reply:
I've reverted the commit until I figure out how to modernize the linter config for examples.

Reply:
>These days I refuse to use the `function` keyword ;-)

Any particular reason? I much prefer it for named functions, because the syntax is just so easy to scan. For callbacks I definitely prefer the fat-arrow notation.

>Perhaps we could have more relaxed (modern) linter configuration for the examples directories?

Yep!

Reply:
The large Typescript project I'm working on doesn't use the `function` keyword, so it seems somewhat anachronistic when I see it! ;-) / 2   Perhaps it's slightly easier to scan... perhaps.  But also more to scan, and it brings up the dynamic scoping of `this`...   Different styles...